frontend/packages/console-plugin-sdk/src/registry.ts

```diff
@@ -17,4 +18,8 @@ export class ExtensionRegistry {
   public getKebabActions() {
     return this.extensions.filter(isKebabActions);
   }
+
+  public getStorageClassProvisioner() {
+    return this.extensions.filter(isStorageClassProvisioner);
+  }
```

Comment:
You dont need registry when you use the `useExtension` hook.

Reply:
+1 
registry is deprecated so we should not add any new code here